### PR TITLE
Remove member access type for cluster templates

### PR DESCRIPTION
**Pull request by mrajashree**
https://github.com/rancher/rancher/issues/21720#issuecomment-521759473

### CODE REVIEW

File: tests/integration/suite/test_clustertemplate.py [hunk 1]
```diff
@@ -309,16 +310,16 @@ def test_revision_creation_permission(admin_mc, remove_resource,
...
              fail_handler=fail_handler(rb_resource))
-    rb_name = name + "-ct-u"
+    rb_name = name + "-ct-a"
     wait_for(lambda: check_subject_in_rb(rbac, 'cattle-global-data',
-                                         user_member.user.id, rb_name),
+                                         user_owner.user.id, rb_name),
              timeout=60,
              fail_handler=fail_handler(rb_resource))
     templateId = cluster_template.id
     # user with accessType=member should not be able to create revision
```
Comment (prachidamle) [inline tests/integration/suite/test_clustertemplate.py]: nit: correction to the comment needed `accessType=owner`

File: tests/integration/suite/test_clustertemplate.py [hunk 2]
```diff
@@ -331,9 +332,9 @@ def test_revision_creation_permission(admin_mc, remove_resource,
     # assign 'clustertemplates-create' role to the user with accessType=member
```
Comment (prachidamle) [inline tests/integration/suite/test_clustertemplate.py]: nit: correction to the comment needed `accessType=owner`

**Comment by prachidamle** [comment]
LGTM

**Comment by alena1108** [comment]
@mrajashree can we drop the member type from the api, so it's not available in schema?

**Comment by mrajashree** [comment]
@alena1108 looks like we cannot remove "member" in schema, since that is an option in the enum for AccessType field from Members struct. This Members is used by MCapp and gdns as well and would affect them too.

**Pull request merged**